libs/ui-toolkit/src/components/tabs/tabs.tsx

```diff
@@ -38,12 +38,22 @@ export const Tabs = ({
     setWrapped(isWrapped(tabsRef.current, menuRef.current));
   });
 
+  const settings = children.find((c) => c?.props.id === activeTab)?.props
```

Comment:
Why not just render the settings within the map of children, then you can access the settings render prop directly